### PR TITLE
wrapping faked name in e function to handle apostrophe

### PR DESCRIPTION
**Pull request by karalan**

### CODE REVIEW

File: tests/Unit/Listeners/SendVoucherHistoryEmailTest.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public function testRequestVoucherHistoryEmail()
         $this->seeEmailWasSent()
             ->seeEmailTo($user->email)
             ->seeEmailSubject('Rose Voucher Payment Records')
-            ->seeEmailContains('Hi ' . $user->name)
+            ->seeEmailContains('Hi ' . e($user->name))
```
Comment (katjam) [inline tests/Unit/Listeners/SendVoucherHistoryEmailTest.php]: What does e do/ stand for and which package is the function from?

Comment (karalan) [reply tests/Unit/Listeners/SendVoucherHistoryEmailTest.php]: It's a laravel helper function https://laravel.com/docs/5.4/helpers#method-e

Comment (katjam) [reply tests/Unit/Listeners/SendVoucherHistoryEmailTest.php]: A nice - is it worth a comment or explicit use of htmlspecialchars? Could be hard to track down in future

Comment (karalan) [reply tests/Unit/Listeners/SendVoucherHistoryEmailTest.php]: I'll add a comment in = )

Comment (katjam) [reply tests/Unit/Listeners/SendVoucherHistoryEmailTest.php]: Sure - no big deal - was curious. Won't hurt. :)

**Pull request merged**